content/en/docs/concepts/workloads/pods/ephemeral-containers.md

```diff
@@ -76,3 +76,270 @@ you can view processes in other containers.
 
 * Learn how to [debug pods using ephemeral containers](/docs/tasks/debug/debug-application/debug-running-pod/#ephemeral-container).
 
+### Kubernetes Ephemeral Container Security
```

Comment:
This is indented under "What's Next" but should probably be its own top level heading,

```
## Ephemeral container security
### Pod Security Policies
### RBAC
etc.
```